tasks/build/package_scripts/pre_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if command -v systemctl >/dev/null && systemctl is-active kibana.service >/dev/null; then
+    systemctl --no-reload stop kibana.service
+fi
+if [ -x /etc/init.d/kibana ]; then
+    if command -v invoke-rc.d >/dev/null; then
+        invoke-rc.d kibana stop
+    elif command -v service >/dev/null; then
```

Comment:
Should you also check here if Kibana is running first, before attempting to stop it?